### ISSUE TITLE
Lock the "Device Mode" for detected mobile devices/viewports

### ISSUE THREAD

**Issue by codeworksdev** [codeworksdev/digital-sandbox#40]
Whenever possible the device mode option should be defaulted (and locked) according to current device specifications. This will prevent the user from losing control of the app if an incompatible device mode is selected(e.g., the user selected "iPad" on an "iPhone").
